Submap map
#transform from previous submap to this one
geometry_msgs/TransformStamped transform
---
bool success
